packages/core/src/geometry/GeometrySystem.js

```diff
@@ -227,6 +234,32 @@ export default class GeometrySystem extends System
         }
     }
 
+    /**
+     * Takes a geometry and program and generates a unique signature for them.
+     *
+     * @param {PIXI.Geometry} geometry to get signature from
+     * @param {PIXI.Program} prgram to test geometry against
+     * @returns {String} Unique signature of the geometry and program
+     * @private
+     */
+    getSignature(geometry, program)
+    {
+        const attribs = geometry.attributes;
+        const shaderAttributes = program.attributeData;
+
+        let id = geometry.id;
+
+        for (const i in attribs)
```

Comment:
1. can we please pass it all to concat?
2. Where is delimiter?

Reply:
tweaked :)